src/Http/Http.Abstractions/src/HttpContext.cs

```diff
@@ -77,16 +79,15 @@ public abstract class HttpContext
 
     private string DebuggerToString()
     {
-        return $"{Request.Method} {Request.Path.Value} {Request.ContentType}"
-            + $" StatusCode = {Response.StatusCode} {Response.ContentType}";
+        return HttpContextDebugFormatter.ContextToString(this, reasonPhrase: null);
     }
 
     private sealed class HttpContextDebugView(HttpContext context)
```

Comment:
What if we add the Endpoint to this as a top level property?

Reply:
I like it.

Reply:
Added `Endpoint` to `HttpContext`:

![image](https://github.com/dotnet/aspnetcore/assets/303201/01213c46-c10c-47b0-b243-09e672f85e8a)